packages/frontend/src/App.svelte

```diff
@@ -62,6 +63,9 @@ onMount(() => {
       <Route path="/disk-images/build/:name/:tag" breadcrumb="Build" let:meta>
         <Build imageName={decodeURIComponent(meta.params.name)} imageTag={decodeURIComponent(meta.params.tag)} />
       </Route>
+      <Route path="/macadam/create/:imageName/:imagePath" breadcrumb="Create Macadam VM" let:meta>
```

Comment:
Nit, but I'd prefer to keep this function instead of having /macadam in the url, e.g. /disk-image/createVM, or maybe we rename both form pages later.